README.md

```diff
@@ -38,6 +38,8 @@ npm run apply-entity-changes
 
 `npm run build`
 
+Note that if you wish to resolve CNAME's, node version 12+ is required.
```

Comment:
Is there a way to run this without resolving cnames?

Reply:
Yes, added some text that also explains the new options that determine whether cnames are resolved.